docs/source/testing/mocking-http-responses.mdx

```diff
@@ -26,7 +26,7 @@ val mockServer = MockServer()
 val apolloClient = ApolloClient.Builder().serverUrl(mockServer.url()).store(store).build()
 
 // Enqueue HTTP responses
-mockServer.enqueue("""{"data": {"random": 42}}""")
+mockServer.enqueueString("""{"data": {"random": 42}}""")
```

Comment:
I always found it confusing that you needed the import to have `enqueue` work (it resolves to the instance function by default). 

`enqueueString` makes it explicit and works better with `enqueueMultipart`

`MockServer.enqueue` is kept as `@DeprecationLevel.Error` for now